pom.xml

```diff
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>net.apnic.rdap</groupId>
     <artifactId>rdap-ingressd</artifactId>
-    <version>1.7.0</version>
+    <version>1.7.1-SNAPSHOT</version>
```

Comment:
Should you be using snapshot instead of a final release?

Reply:
no idea why github is picking this up as a change. None of the commits have this file and it is currently set to this value in develop.

Bug in github perhaps?

Reply:
NM was user error